components/scientific/openblas/manifests/sample-manifest.p5m

```diff
@@ -22,8 +23,28 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
+file path=opt/OpenBLAS/include/openblas/cblas.h
+file path=opt/OpenBLAS/include/openblas/f77blas.h
```

Comment:
There is something wrong here: We don't deliver files to /opt!

Reply:
It looks like the new version tries to install the same files twice - In /opt and in their correct location.
I already removed the /opt entries from openblas.p5m and thought that should be sufficient.
I have no idea how I should stop it from trying to install to /opt.
When I remove the PREFIX from the install args, it installs *only* to /opt and in the current form it wants to install to both.

Reply:
It is sufficient. It's just irritating that sample-manifest contains theses files. Something seems to be wrong. Maybe you can try to find the reason for it and fix it.

Reply:
I moved DROP_STATIC_LIBRARIES a few lines up and build everything again, now the /opt entries are gone.
I can't really imagine that this is the reason for the difference (maybe it somehow didn't correctly clean the prototype directory before or something like that) but at least it's correct now.